controllers/controllers.errors.js

```diff
@@ -1,3 +1,13 @@
+const handlesPsqlErrors = (err, req, res, next) => {
+    if(err.code === "22P02") {
+        res.status(400).send({ msg: "Bad Request" });
+      } else if (err.code === "23503") {
+        res.status(404).send({ msg: "Path Not Found" });
```

Comment:
Here is where, if you would like to, change the message to something more like "not found" or "resource not found" - removing the word "path"